ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.2.0
 -----
+- `tools/file_packager` no longer generates (or requires) any "pre-js" code when
+  running in `--embed-file` mode.  Instead the embedded files are loaded as
+  static constructor time.
```

Comment:
Given this change, should we recommend embedding over preloading in all our docs?

Reply:
I think that should be the recommendation for most users yes, since embedding is very efficient.  

The only advantages of preloading that I can think of are:
1. It allows the download of the files to happen in parallel with the download of the wasm file
2. If possibly allows the files to change or be updated without updating the build.. (Not sure about that one)

However that recommendation doesn't really change with this specific PR, I think it changed when we started embedding the data in the wasm.